x-pack/plugins/alerting/server/types.ts

```diff
@@ -74,13 +74,14 @@ export interface RuleExecutorServices<
   Context extends AlertInstanceContext = AlertInstanceContext,
   ActionGroupIds extends string = never
 > {
-  searchSourceClient: ISearchStartSearchSource;
```

Comment:
would be better to leave the original order, so we only see the actual changes; I had to check each add/delete to make sure only the order changed.